docs/guides/stonesense-art-guide.rst

```diff
@@ -0,0 +1,64 @@
+.. _stonesense-art-guide:
+
+Stonesense art creation guide
+=============================
+
+Understanding sprites
+---------------------
+
+Understanding how Stonesense deals with sprites is central to anyone who wishes to modify the content. The
+scheme is not very complicated, and this guide will give a short introduction to how they work. With the
+exception of floors, which we will discuss later, all sprites are 32x32 pixels big and come in groups known
+as Sprite Sheets. All sprites are loaded and rendered in 32-bit full-color PNGs. The image files should have
+a transparent background but pure magenta (RGB: 255,0,255) is also treated as transparent.
+
+
+.. image:: ../images/stonesense-sprite-sample.png
+    :align: left
+
+Here's an example of a typical Stonesense sprite.
+
+Note that, in order not to conflict with neighboring sprites, a sprite must actually be within a smaller
+area than its 32x32 block, typically 30x30.
```

Comment:
I don't quite understand this. Why would a sprite that uses its full allocated space "conflict" with its neighboring sprites? The DF sprite sheets have sprites up to their boundary pixels, adjacent to the next sprite over, and there aren't any issues there. What problems does Stonesense have with larger sprites?

Edit: Does this have to do with the layout requirements for isometric perspective? If so, I think that needs to be introduced earlier. Also, is it actually 30x30, or is it the boundary defined by the volume template below?